src/sw/helpers/ServiceWorkerUtilHelper.ts

```diff
@@ -33,4 +33,29 @@ export default class ServiceWorkerUtilHelper {
     }
     return availableWorker;
   }
+
+  // Allows waiting for the service worker registration to become active.
+  // Some APIs, like registration.pushManager.subscribe, required it be active
+  // otherwise it throws.
+  static waitForServiceWorkerToActive(
+    registration: ServiceWorkerRegistration,
+  ): Promise<void> {
+    return new Promise((resolver) => {
```

Comment:
Nit: typically this is just `resolve`

Reply:
hmm, prettier wants `(resolver)` instead of `resolver` here. I get this error when I try to change it to your suggestion:
```
 src/sw/helpers/ServiceWorkerUtilHelper.ts:43:24
  ⚠  43:24  Replace resolver with (resolver)  prettier/prettier
```

Reply:
I meant `resolve` over `resolver`

Reply:
ah yes, my bad🤦. fixed